python/pipeline/preprocess.py

```diff
@@ -130,6 +130,21 @@ def estimate_num_components_per_slice(self):
 
             return int(round(num_components))
 
+        def estimate_neuron_size_in_pixels(self):
+            """ Estimates the size of a neuron in the scan (in pixels). Assumes neurons
+             are 15 x 15 microns."""
+            neuron_size_in_microns = 15 # assumption
+
+            # Calculate size in pixels
+            um_width, px_width = (Prepare.Galvo() & self).fetch1['um_width', 'px_width']
+            one_micron_in_pixels = px_width / um_width
```

Comment:
There is a possibility of deviations from isometry of pixels:  `um_width/px_width` may differ from `um_height/px_height`.  Perhaps we should at least check that the deviation is not serious or handle it somehow if it is.